protocol/contracts/templegold/SpiceAuction.sol

```diff
@@ -358,6 +358,8 @@ contract SpiceAuction is ISpiceAuction, AuctionBase, ReentrancyGuard, Initializa
         uint256 amount = epochInfo.totalAuctionTokenAmount;
         epochInfo.totalAuctionTokenAmount = 0;
         _totalAuctionTokenAllocation[auctionToken] -= amount;
+        // mark epoch as redeemed
+        redeemedEpochs[epochId] = true;
```

Comment:
Small bug that crept in? Was there a test for this?

Reply:
I updated the test at `protocol/test/forge/unit/templegold/SpiceAuction.t.sol`